src/chrome/local-runtime.ts

```diff
@@ -208,6 +211,34 @@ export default class LocalRuntime {
     }
   }
 
+  // Returns the S3 url or local file path
+  async returnPDF(): Promise<string> {
+    const data = await pdf(this.client)
+
+    // check if S3 configured
+    if (process.env['CHROMELESS_S3_BUCKET_NAME'] && process.env['CHROMELESS_S3_BUCKET_URL']) {
+      const s3Path = `${cuid()}.pdf`
```

Comment:
So we save both screenshots and pdfs to the same bucked and to the same folder.

What do you think of putting them like `bucket/screenshots/${cuid()}.png` and `bucket/pdfs/${cuid()}.pdf` ?